app/rest/model/src/test/java/io/syndesis/model/extension/ExtensionSchemaValidationTest.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
```

Comment:
2017?

Reply:
The syndesis license header plugin requires 2016. We should update all headers (better to 2018 :D) in another PR.